docs/UserGuide.md

```diff
@@ -47,110 +49,161 @@ Format: `help`
 
 > Help is also shown if you enter an incorrect command e.g. `abcd`
 
-### 2.2. Adding a person: `add`
+### 2.2. Adding a task/event : `add`
+
+Adds a task/event to the address book<br>
+
+Task: 
+
+Format: `add task TASK_NAME [dl/DEADLINE] [des/DESCRIPTION] [t/TAG]...`
+
+> Tasks can have a deadline
+> Tasks can have a description
+> Tasks can have any number of tags (including 0)
+
+
+Examples:
+* `add Do laundry dl/2017-3-1-2359 `
+* `add Study for midterms dl/2017-3-2-2359 des/CS2103 at lt7 `
+* `add Buy milk for baby dl/2017-3-3-2100 des/yaas milk t/family`
+* `add Create user story dl/2017-4-1-1300 t/work t/computing` 
+
+Event: 
+
+Format: `add event EVENT_NAME st/START_TIME et/END_TIME [des/DESCRIPTION] [t/TAG]...`
+
+> * Event can have description
+> * Events can have any number of tags (including 0)
+
+
+Examples:
+* `Add Midterms st/2017-3-1-1300 et/2017-3-1-1500 des/CS2103 t/school`
+* `Add Kaili Birthday st/2017-8-30-0000 et/2017-8-31-2359 t/friends`
+* `Add Gym legs day st/2017-3-4-0600 et/2017-3-4-0700 t/health`
+
+### 2.3. Finding all tasks/events containing any keyword in their name or tag : `find`
+
+
+Finds tasks/events whose names contain any of the given keywords.<br>
 
-Adds a person to the address book<br>
-Format: `add NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...`
+Format: `find KEYWORD [MORE_KEYWORDS]`
 
-> Persons can have any number of tags (including 0)
+> * The search is case sensitive. e.g homework will not match Homework
```

Comment:
think about usability. how can you make it easier for the user to find something. consider case insensitive search